invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -349,9 +361,14 @@ def get_descriptions(self, obj):
 
         additional_descriptions = metadata.get("additional_descriptions", [])
         for add_desc in additional_descriptions:
+            # TODO: Update when we know how to store Datacite values in vocabs
+            description_type_id = add_desc.get("type").get("id")
+            description_type = self._read_description_type(
+                description_type_id
+            ).to_dict().get("title")
             description = {
                 "description": add_desc["description"],
-                "descriptionType": add_desc["type"].capitalize()
+                "descriptionType": description_type.get(get_locale().language)
```

Comment:
This is not correct. It should be taken from the DataCite type, but I should have fixed it in #704